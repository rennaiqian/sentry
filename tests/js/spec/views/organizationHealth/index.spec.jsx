import React from 'react';
import {mount} from 'enzyme';

import OrganizationHealth from 'app/views/organizationHealth';

describe('OrganizationHealth', function() {
  it('updates projects', function() {
    let wrapper = mount(<OrganizationHealth />);

    expect(wrapper).toMatchSnapshot();
  });
});
