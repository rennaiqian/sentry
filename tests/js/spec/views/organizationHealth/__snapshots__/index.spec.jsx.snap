// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrganizationHealth renders 1`] = `
<OrganizationHealthOverview>
  <div>
    Overview
  </div>
</OrganizationHealthOverview>
`;
